Sources/Fluent/Schema/Builder.swift

```diff
@@ -154,14 +200,66 @@ extension Builder {
     // MARK: Relations
 
     public func parent<E: Entity>(
+        field name: String = E.foreignIdKey,
         _ entity: E.Type = E.self,
         optional: Bool = false,
-        unique: Bool = false
+        unique: Bool = false,
+        constraint fkCreation: ForeignKeyCreation = .auto
     ) {
         foreignId(
+            name,
             for: E.self,
             optional: optional,
-            unique: unique
+            unique: unique,
+            constraint: fkCreation
+        )
+    }
+    
+    public func owner<E: Entity>(
```

Comment:
It should be up to the user to add these type-aliases if they want. Providing multiple ways to trigger the same function with different names could confuse people.